src/angular/11-declarative-state/mergeMap.html

```diff
@@ -0,0 +1,47 @@
+<script src="https://cdnjs.cloudflare.com/ajax/libs/rxjs/6.3.3/rxjs.umd.min.js"></script>
+<script type="module">
+  const { of, interval, zip } = rxjs;
+  const { map, mergeMap, delay } = rxjs.operators;
+
+  // returns a stream that emits the price of a given car after a 500ms delay. this is how a request might operate
+  function pseudoPriceRequest(carName) {
+    if (carName === 'mustang') {
+      return of(40000).pipe(delay(500));
+    } else if (carName === 'camaro') {
+      return of(38000).pipe(delay(500));
+    }
```

Comment:
Not a blocker, but should we handle if carName isn't `mustang` or `camaro`? Like throw or something

Reply:
it is just an example of mergeMap. I don't know, no other examples handle errors like that

Reply:
reviewing the example - would a shopping cart with item total be a better way to show mergeMap?

Reply:
so actually there are no changes here other than file name change from flatMap to mergeMap and of course the method being used from flatMap to mergeMap.
I'm fine if we want to change the example but could it be done in a separate issue?

Reply:
https://bitovi.atlassian.net/browse/NGACAD-80